tools/src/main/scala/scala/scalanative/build/NativeLib.scala

```diff
@@ -185,4 +178,12 @@ private[scalanative] object NativeLib {
       case true  => Some(path)
       case false => None
     }
+
+  private def makeDirPath(path: Path, elems: String*): String = {
+    val pathSep = if (Platform.isWindows) raw"\\" else File.separator
```

Comment:
This is needed because it is inside a `glob` which is like a regex?

Reply:
Yes, I think that glob in java is also implemented based on regex. 
Without this, we would not be able to correctly find all native libs.